aopy/preproc/base.py

```diff
@@ -535,6 +535,7 @@ def get_trial_segments_and_times(events, times, start_events, end_events):
         times (nt): times vector
         start_events (list): set of start events to match
         end_events (list): set of end events to match
+        repeating_start_events (bool): whether the events used as start events may occur multiple times within one segment
 
     Returns:
         tuple: tuple containing:
```

Comment:
would be helpful to include two examples here, one with and one without the `repeating_start_events` option, with an explanation of which events are repeating and why (multiple random targets in a trial, for instance)